editor/src/components/editor/canvas-toolbar.tsx

```diff
@@ -489,8 +490,31 @@ export const CanvasToolbar = React.memo(() => {
                 onClick={selectInsertMenuPane}
               />
             </Tooltip>
+            {/* Live Mode */}
```

Comment:
?

Reply:
This is consistent with the rest of the code in the file, the separate sections are broken up by comments such as `{/* Insert Mode */}`, and `{/* Edit Mode submenus */}`. https://github.com/concrete-utopia/utopia/pull/4195/commits/e1ba2ea571479ab8cf8d16e49eff10f030f8b708 moves the comment to the right place though